engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetViewportSizeOf.java

```diff
@@ -86,6 +84,9 @@ public void update() {
 
     @Override
     public String toString() { // TODO: used for logging purposes at the moment, investigate different methods
+        if (isTheDefaultInstance()) { // necessary since default instance is generated before setFrameBuffersManager
+            fbo = frameBuffersManager.getFBO(fboName);
+        }
```

Comment:
Just to be clear, the viewport state change should also stay in the log, when the intermediate list is printed out. It just doesn't need to printout the width and height information, only the fboName. I'm not sure if I was clear on this.